package.xml

```diff
@@ -1,21 +1,16 @@
 <?xml version="1.0"?>
 <package format="2">
   <name>mrover</name>
-  <version>0.0.0</version>
-  <description>The mrover package</description>
-
-  <!-- One maintainer tag required, multiple allowed, one person per tag -->
-  <!-- Example:  -->
-  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
-  <maintainer email="ashwin@todo.todo">ashwin</maintainer>
+  <version>1.0.0</version>
+  <description>MRover ROS Package</description>
```

Comment:
lowkey "The mrover package" has a nice ring to it...

Reply:
So you want it to be `<description>The Mrover Package</description>`?

Reply:
Lol nbd